include/linux/slaunch.h

```diff
@@ -495,18 +495,61 @@ static inline int tpm20_log_event(struct txt_heap_event_log_pointer2_1_element *
 }
 
 struct sl_header {
-	u16 lz_offset;
-	u16 lz_length;
+	u16 lz_entry_point;
+	u16 bootloader_data_offset;
+	u16 lz_info_offset;
```

Comment:
What is this field for? As I understand it the bootloader_data_offset points to a block where the various tag structures below can be found but I don't know what this field points to.

Reply:
This is an offset to a structure holding information about the LZ: UUID, version, MSB key hash, in the future maybe capabilities flag, if such need arises. Designed mostly to get information from LZ image in bootloader, but maybe kernel would like to use it too.

Reply:
Ack thanks